static/gsApp/components/gsBanner.tsx

```diff
@@ -979,6 +980,10 @@ class GSBanner extends Component<Props, State> {
     let overquotaPrompt: React.ReactNode;
     let eventTypes: EventType[] = [];
 
+    if (prefersStackedNav()) {
```

Comment:
Might be good to drop a comment here to explain why we don't render in this circumstance